ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,226 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb106" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Ṣoma dǝggʷā</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="NV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10088581j">BnF Éthiopien d'Abbadie 106</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1ra" to="58va"/>
+                            <title type="complete" ref="LIT2361somade">Dǝggʷā chants for Lent period</title>
+                            <note> Used in the edition <bibl> <ptr target="bm:Velat1966Soma"></ptr></bibl>
```

Comment:
unless Velat describes the **manuscript** itself - this information should be recorded with the work record and not here

so in `LIT2361somade `you should assign `xml:id` to the `witness `in the witnesses list and add a `corresp `to the `edition `bibliography to show which manuscripts were used in the edition

Reply:
Can you please point to a right example? I fail to find it myself. Thank you!

Reply:
Example 1 https://betamasaheft.eu/Guidelines/?id=bibliographic-references